code/iaas/allocator/src/main/java/io/cattle/platform/allocator/constraint/AffinityConstraintDefinition.java

```diff
@@ -7,8 +7,8 @@
         EQ("==", "{eq}"),
         NE("!=", "{ne}");
 
-        String envSymbol;
-        String labelSymbol;
+        public String envSymbol;
```

Comment:
As a part of my refactoring that got merged to master recently, I've introduced getters for retrieving these values (instead of "public" access modifier). Would it be possible for your code to use getters as well, so we are consistent.